docs/classad-attributes/collector-classad-attributes.rst

```diff
@@ -1,74 +1,53 @@
 Collector ClassAd Attributes
 ============================
 
-
 :index:`Collector attributes<single: Collector attributes; ClassAd>`
-:index:`ActiveQueryWorkers<single: ActiveQueryWorkers; ClassAd Collector attribute>`
 
-``ActiveQueryWorkers``:
+:classad-attribute:`Collector attributes`
     Current number of forked child processes handling queries.
 
-:index:`ActiveQueryWorkersPeak<single: ActiveQueryWorkersPeak; ClassAd Collector attribute>`
-
-``ActiveQueryWorkersPeak``:
+:classad-attribute:`ActiveQueryWorkersPeak`
     Peak number of forked child processes handling queries since
     collector startup or statistics reset.
 
-:index:`PendingQueries<single: PendingQueries; ClassAd Collector attribute>`
-
-``PendingQueries``:
+:classad-attribute:`PendingQueries`
```

Comment:
Could we move this and `PendingQueriesPeak` down a ways and alphabetize this page?